bot/plotting/comparison_generator.py

```diff
@@ -127,15 +130,36 @@ def comparison_generator(
 
                 is_experiment = False
 
+                # vary "Current function [A]" and "Ambient temperature [K]"
```

Comment:
Not sure if this still happens, but you shouldn't vary "Current function [A]" if you are using an experiment since the experiment will override the current function anyway.

Reply:
Yes, a recent [tweet](https://twitter.com/battbot_/status/1416841242323922952) had this issue. I will fix this